nifstd/resources/sparc_terms2.txt

```diff
@@ -1,1078 +1,1090 @@
-        Central Oculomotor NucleusὊ9UBERON:0002960
-        Medial Oculomotor NucleusὊ9UBERON:0002975
-        Parvocellular Oculomotor NucleusὊ9UBERON:0002141
-        Ventral Oculomotor NucleusὊ9UBERON:0002934
-        Caudal Central Oculomotor NucleusὊ9UBERON:0002957
-        Anterior Median Oculomotor NucleusὊ9UBERON:0002701
-        Intermediate Oculomotor NucleusὊ9UBERON:0002970
-        Dentatothalamic TractὊ9UBERON:0002594
-        Interstitial Nucleus Of CajalὊ9UBERON:0002551
-        Midbrain Reticular FormationὊ9UBERON:0002639
-        Pedunculopontine Tegmental NucleusὊ9UBERON:0002142
-        Subcuneiform NucleusὊ9UBERON:0002700
-        Cuneiform NucleusὊ9UBERON:0002696
-        Central Tegmental Tract Of MidbrainὊ9UBERON:0002585
-        Midbrain Raphe NucleiὊ9UBERON:0007412
-        Pallidotegmental FasciculusὊ9UBERON:0002671
-        Dorsal Tegmental NucleusὊ9UBERON:0002143
-        Red NucleusὊ9UBERON:0001947
-        Parvocellular Part Of Red NucleusὊ9UBERON:0002938
-        Magnocellular Part Of Red NucleusὊ9UBERON:0002936
-        Central Gray Substance Of MidbrainὊ9UBERON:0003040
-        Root Of Trochlear NerveὊ9UBERON:0002618
-        Mesencephalic Nucleus Of Trigeminal NerveὊ9UBERON:0001718
-        Peripeduncular NucleusὊ9UBERON:0002144
-        Spinothalamic Tract Of MidbrainὊ9UBERON:0002609
-        Decussation Of Superior Cerebellar PeduncleὊ9UBERON:0002588
-        Medial Lemniscus Of MidbrainὊ9UBERON:0002658
-        Habenulo-Interpeduncular Tract Of MidbrainὊ9UBERON:0023740
-        Dorsal Raphe NucleusὊ9UBERON:0002043
-        Ventral Nucleus Of Posterior CommissureὊ9UBERON:0002044
-        Ventral Tegmental AreaὊ9UBERON:0002691
-        Ventral Tegmental DecussationὊ9UBERON:0002615
-        Medial Pes LemniscusὊ9UBERON:0002662
-        Ventral Tegmental NucleusὊ9UBERON:0002438
-        Medial Longitudinal Fasciculus Of MidbrainὊ9UBERON:0002660
-        Mesencephalic Tract Of Trigeminal NerveὊ9UBERON:0002666
-        Interpeduncular NucleusὊ9UBERON:0002145
-        Trochlear NucleusὊ9UBERON:0002722
-        Substantia NigraὊ9UBERON:0002038
-        Substantia Nigra Pars CompactaὊ9UBERON:0001965
-        Substantia Nigra Pars ReticulataὊ9UBERON:0001966
-        Substantia Nigra Pars LateralisὊ9UBERON:0002995
-        Cerebral CrusὊ9UBERON:0002631
-        Midbrain TectumὊ9UBERON:0002314
-        Posterior CommissureὊ9UBERON:0000936
-        Corticotectal TractὊ9UBERON:0002556
-        Inferior ColliculusὊ9UBERON:0001946
-        Subcommissural OrganὊ9UBERON:0002139
-        Parabigeminal NucleusὊ9UBERON:0002140
-        Pretectal RegionὊ9UBERON:0001944
-        Olivary Pretectal NucleusὊ9UBERON:0002565
-        Nucleus Of Optic TractὊ9UBERON:0002996
-        Principal Pretectal NucleusὊ9UBERON:0002572
-        Sublentiform NucleusὊ9UBERON:0002578
-        Anterior Pretectal NucleusὊ9UBERON:0034918
-        Superior ColliculusὊ9UBERON:0001945
+        retrorubral area of midbrain reticular nucleusὊ9UBERON:0011172
```

Comment:
Looks like the delimiter is still here, use tab and switch between loading from file and passing to yaml.loads.